### DISCUSSION TITLE
Use a modern implementation of Ahocorasick

### DISCUSSION THREAD

**Discussion by brandonpittman** [vanilla-extract-css/vanilla-extract#1196] [Ideas]
While implementing a Box component for Qwik, I ran into a problem with Ahocorasick. The implementation VE uses doesn't export anything and that's creating an error (at least in Qwik when conditionally rendering the Box component). 

Can we just bring this function into the repo or use [a more modern implementation](https://github.com/sonofmagic/modern-ahocorasick/blob/main/src/index.ts)?

**Comment by mattcompiles** [comment]
Happy to accept a PR mate 🙂

**Reply by brandonpittman** [reply]
#1197 There you go.
